src/main/java/log_out/use_case/LogOutRequestModel.java

```diff
@@ -5,9 +5,10 @@ public class LogOutRequestModel {
 
     /**
      * Initialize a log_out Request
-     * @param logOutRequest A string to indicate a logout request has been made
+     *  A string to indicate a logout request has been made
      */
-    public LogOutRequestModel(String logOutRequest){
+
+    public LogOutRequestModel(){
```

Comment:
again, is this supposed to be empty?

Reply:
Yeah, for logging out theres not really any data that needs to be processed the parameter that was there before was never being used. It works the same without it